lib/index.js

```diff
@@ -343,3 +362,26 @@ function _updateSessionAuthenticationData(
   }
   return data;
 }
+
+function _getRandomValues() {
```

Comment:
Does this need to use webcrypto API?  Probably easier to use only `randomFill` here.

Details: Bedrock code is Node.js 16+ so if wanting to use `getRandomValues` then could drop `randomFill` support.  Just to make it confusing, Node 16 uses `crypto.webcrypto.getRandomValues`, 18 uses `crypto.webcrypto.getRandomValues` or `crypto.getRandomValues`, and 20 use `crypto.getRandomValues` and doesn't even have `crypto.webcrypto`.  So I'd like to think it could be `return crypto?.getRandomValues ?? crypto.webcrypto.getRandomValues`, but I don't know how the `bind` thing would work with that.

Reply:
Luckily, we won't have to answer any of these questions / do anything here because I don't think we need the random bytes at all with my above suggestion :). We'll just call the node.js crypto hmac API instead.

Reply:
> I think what we want to do instead is set a not-so-secret hmac key in the config that can be overwritten in production. Then we should hmac the given username / email (whatever the input is) and use that to produce a consistent salt to use in what gets returned. We should do that on every request -- concurrently with getting the token from the database. This should help minimizing timing differences.

I generate the salt using hmac. Using this, we can get a constant salt with given username or email.
So removed `_getRandomValues` function.

Reply:
> Does this need to use webcrypto API? Probably easier to use only `randomFill` here.
> 
> Details: Bedrock code is Node.js 16+ so if wanting to use `getRandomValues` then could drop `randomFill` support. Just to make it confusing, Node 16 uses `crypto.webcrypto.getRandomValues`, 18 uses `crypto.webcrypto.getRandomValues` or `crypto.getRandomValues`, and 20 use `crypto.getRandomValues` and doesn't even have `crypto.webcrypto`. So I'd like to think it could be `return crypto?.getRandomValues ?? crypto.webcrypto.getRandomValues`, but I don't know how the `bind` thing would work with that.

>Luckily, we won't have to answer any of these questions / do anything here because I don't think we need the random bytes at all with my above suggestion :). We'll just call the node.js crypto hmac API instead.

I should note that this block came from [bedrock-authn-token](https://github.com/digitalbazaar/bedrock-authn-token/blob/main/lib/pbkdf2.js#L166-L177). The test matrix over there runs on node 14,16, and 18, if bedrock no longer support version of node less than 16, perhaps it could get cleaned up there, or maybe an issue filed if it needs a rework?

Reply:
Added issue.